SharedUtils/src/main/java/com/braintreepayments/api/BTHttpResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.braintreepayments.api
+
+class BTHttpResponse constructor(
```

Comment:
```suggestion
class HttpResponse constructor(
```
We should get rid of the "BT" prefixes. Packages in both Java and Kotlin remove the need for us to prefix our class names.

Reply:
➕ 1

Reply:
Fix: [15c60c1](https://github.com/braintree/braintree_android/pull/1032/commits/15c60c180911a84ad08427c09531b8c5221056b4)